#!/bin/sh /usr/share/dpatch/dpatch-run
## 11-gcc-4.3.dpatch by Michael Meskes <meskes@debian.org>
##
## DP: Some changes needed by gcc-4.3

@DPATCH@

diff -Naurp virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Main/VMMDevInterface.cpp virtualbox-ose-1.6.0-dfsg/src/VBox/Main/VMMDevInterface.cpp
--- virtualbox-ose-1.6.0-dfsg.orig/src/VBox/Main/VMMDevInterface.cpp	2008-04-30 14:05:49.000000000 +0000
+++ virtualbox-ose-1.6.0-dfsg/src/VBox/Main/VMMDevInterface.cpp	2008-05-09 09:31:08.000000000 +0000
@@ -31,6 +31,7 @@
 #include <VBox/VBoxGuest.h>
 #include <VBox/shflsvc.h>
 #include <iprt/asm.h>
+#include <stdio.h>
 
 #ifdef VBOX_HGCM
 #include "hgcm/HGCM.h"
